cmd/serve.go

```diff
@@ -66,6 +65,11 @@ func (s *Serve) GetCmd() *cobra.Command {
 				return
 			}
 
+			if err != nil {
```

Comment:
You can remove this code since there is no build-delay in the serve command. So no need to check for this error.